controllers/dspipeline_params.go

```diff
@@ -245,7 +247,7 @@ func (p *DSPAParams) SetupObjectParams(ctx context.Context, dsp *dspa.DataScienc
 		// populated with defaults.
 		setStringDefault(config.MinioImage, &p.Minio.Image)
 		setStringDefault(config.MinioDefaultBucket, &p.Minio.Bucket)
-		setStringDefault(config.MinioPVCSize, &p.Minio.PVCSize)
+		p.Minio.PVCSize = resource.MustParse(config.MinioPVCSize)
```

Comment:
2 things here: 

1. we want to only set default if user has not specified a PVC Size in the CR, atm we always use default, see `setStringDefault`, we may want something similar for resource struct 

2. We also want to do something similar for mariaDB. At the moment it will still work without something like this for mariadb because [we have default via kubebuilder annotation](https://github.com/opendatahub-io/data-science-pipelines-operator/pull/35/files#diff-4606cde41444090b0c00d3e515fee3ac2e1ce6de20de579165193a861175a6acR138). If we want to rely on this default, then we should be consistent and take this out of the minio logic here. If we want to ensure we check for defaults in code, and not just rely on k8s to ensure defaults on it send, then add default check in mariadb.

Reply:
changed it to rely on the kubebuilder annotation